.github/workflows/pull_request.yml

```diff
@@ -14,6 +14,51 @@ on:
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
+  kind_setup:
```

Comment:
It's hard to find a good name, but this seems to "small". It's not just about the kind cluster but about building + deploying forklift...

Reply:
sooo just `ci_setup` or `setup` to make it more obscure? Don't want to list everything this name should be general

Reply:
Yes. That's why I named it "do it" before ;-)